Date 20-01-06

W. Zac Stephens

# Background
Process:
1. Base call (optional, would usually do locally with Guppy)
2. Clean reads. QC/nanopolish
3. Remove human mapping reads, mouse mapping reads (later, check both of these against microbial)
4. Assembly-based metagenomes
5.


# Method
1. Build container with
   1. Guppy
   2. nanopolish
   3. SqueezeMeta

# Input Files
1. Reference database?
2.

``` {r}
get.
```
#
